vmdb/app/models/miq_provision_microsoft_workflow.rb

```diff
@@ -0,0 +1,63 @@
+class MiqProvisionMicrosoftWorkflow < MiqProvisionInfraWorkflow
+  def self.default_dialog_file
+    'miq_provision_dialogs'
+  end
+
+  def self.default_pre_dialog_file
+    'miq_provision_dialogs_pre'
+  end
+
+  def self.encrypted_options_fields
+    super + [:sysprep_password, :sysprep_domain_password]
```

Comment:
You do not have these sysprep fields defined in your dialog so you do not need them here.  The parent class defines the :root_password field which is the only one currently in the dialog so this method can be removed as well.